examples/tutorials/basics/regions.py

```diff
@@ -74,7 +73,8 @@
 )
 fig.show()
 
```

Comment:
The number of blank lines above the `# %%` separator is inconsistent. Maybe we should have only one blank line above `# %%`, although it really doesn't matter much.

Reply:
This is because there are two blank lines in case of a following subsection, otherwise only one blank is used. This is the concept in some of the tutorials. For me, this makes it easier to read the script, because it gives a bit more structure, especially for longer ones. Thus, I tried / decided to use this consistently across all examples. But as you mentioned, this doesn't matter much, I can change it to have only one blank line before subsections.

Reply:
> This is because there are two blank lines in case of a following subsection, otherwise only one blank is used.

OK. That makes sense to me.